conf.py

```diff
@@ -25,24 +30,19 @@
 }
 
 # List of patterns (relative to source dir) to ignore when looking for source files.
+include_patterns = [
+    # Required for Sphinx
+    "contents.rst",
+    # PEP files
+    "pep-????.???",
+    # PEP ancillary files
+    "pep-????/*.rst",
```

Comment:
```suggestion
    "pep-[0-9][0-9][0-9][0-9]/*.rst",
```

More precise like the above; right now any rst files in a directory named `pep-` and four characters will be picked up, e.g. `pep-help/`. `pep-temp/`, etc.  This only detects directories that match the format in PEP 1/PEP 12.